ci/properties/openshift.properties.json

```diff
@@ -3,5 +3,5 @@
     "description": "Build a Docker-based project and deploy it to OpenShift.",
     "creator": "Red Hat",
     "iconName": "openshift",
-    "categories": [ "Dockerfile" ]
+    "categories": ["Dockerfile","Deployment" ]
```

Comment:
Lets keep `Deployment` as first entry. It is not a requirement as such but it just keeps the readability easier.

Reply:
the first category listed is considered the "primary category" and hence keeping "Dockerfile" first in the list to maintain the status quo . [Link](https://github.com/github/github/blob/1bb860fea968de947cb4b740aab65f67404a01c9/app/view_models/actions/starter_workflows_view.rb#L156)

Reply:
Got it. May be we should fix this with the new UI so that we do not depend on the position to decide primary language for a template.